### PR TITLE
Fix alignment of reports

### LINKED ISSUE #3002: [Bug]: Custom reports do not align visually with default reports
### Verified issue does not already exist?

- [X] I have searched and found no existing issue
- [X] I will be providing steps how to reproduce the bug (in most cases this will also mean uploading a demo budget file)

### What happened?

When you have two or more custom reports, there is a noticeable difference in the size/spacing of the reports between the custom reports and the default out-of-the-box reports.

Screenshot from 2024-07-07 21-14-12

### Where are you hosting Actual?

Other

### What browsers are you seeing the problem on?

Firefox

### Operating System

Linux

### PR DESCRIPTION
**Pull request by JukeboxRhino**
Resolves #3002

## Before:
image
image
image

## After:
image
image
image

### CODE REVIEW

**Review by MatissJanis — APPROVED** [review]

File: packages/desktop-client/src/components/reports/reports/CustomReportListCards.tsx [hunk 1]
```diff
@@ -47,9 +47,9 @@ export function CustomReportListCards({
   const payees = usePayees();
   const accounts = useAccounts();
   const categories = useCategories();
-  const { isNarrowWidth } = useResponsive();
   const [_firstDayOfWeekIdx] = useLocalPref('firstDayOfWeekIdx');
   const firstDayOfWeekIdx = _firstDayOfWeekIdx || '0';
+  const { isNarrowWidth } = useResponsive();
```
Comment (carkom) [inline packages/desktop-client/src/components/reports/reports/CustomReportListCards.tsx:52]: nit - does this need to be moved?

**Review by carkom — APPROVED** [review]
This LGTM! One small nit, not a big issue though.

File: packages/desktop-client/src/components/reports/reports/CustomReportListCards.tsx (hunk 1 above)
Comment (JukeboxRhino) [reply packages/desktop-client/src/components/reports/reports/CustomReportListCards.tsx:52]: Sorry, I don't quite follow. Do you mean moved to a different spot in the file, or moved to another file? It passes lint so I don't think it's the former, and if it's the latter I'm using it here to calculate element flex-basis in desktop vs mobile screen sizes

Comment (JukeboxRhino) [reply packages/desktop-client/src/components/reports/reports/CustomReportListCards.tsx:52]: @carkom Let me know if there's anything I need to do to get this merged

**Comment by youngcw** [comment]
Looks like that nit really is more of a curiosity.  Merging

**Pull request merged**